index.html

```diff
@@ -307,22 +355,15 @@ <h3 class="contact-title">Contact me</h3>
           </li>
           <li>
               <label for="msg"></label>
-              <textarea id="msg" name="user_message" maxlength="500" cols="30" rows="10" required placeholder="Write your message here"></textarea>
+              <textarea class="text-area" name="message" maxlength="500" cols="30" rows="10" required placeholder="Write your message here"></textarea>
         </ul>
```

Comment:
- [ ] You are almost there but small chance should be done please implement it as contact form cover the whole width if you see the design please implement it and also manage the spacing between paragraphs and form
### Your design
![your](https://user-images.githubusercontent.com/81727988/194684465-518e67c9-c165-4d22-9017-ab396711abd2.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/194684493-7b66baa1-4ac8-4d2d-b679-129836dc491d.png)